api/src/schema/arrivals.graphql

```diff
@@ -74,6 +74,28 @@ extend type Member {
   isArrivalsCounterForStream: [Stream!]!
     @relationship(type: "COUNTS_ARRIVALS_FOR", direction: OUT)
 }
+
+extend type Oversight {
+  graduatedBacentas: [Bacenta!]!
+    @cypher(
+      statement: "MATCH (this)-[:HAS*5]->(bacentas:Graduated) RETURN bacentas"
+    )
+  icBacentas: [Bacenta!]!
+    @cypher(statement: "MATCH (this)-[:HAS*5]->(bacentas:IC) RETURN bacentas")
+  icBacentaCount: Int
+    @cypher(
+      statement: "MATCH (this)-[:HAS*5]->(bacentas:IC) RETURN COUNT(DISTINCT bacentas)"
+    )
+  activeIcBacentaCount: Int
+    @cypher(
```

Comment:
@dabick14 Make sure you add `:Bacenta` to this. So that closed bacentas don't pop up in this query.

Please do this through out.

Reply:
This has not been resolved

Reply:
@dabick14 Please do what I'm saying